src/LSTM/index.js

```diff
@@ -146,7 +158,11 @@ class LSTM {
         generated += mapped;
       }
     });
-    return generated;
+    return {
```

Comment:
yes we could do this. Actually maybe we don't need to return `state` since it's a class property? And maybe we could make `probabilities` a class property too? So `generate` always returns a sample, and anyone interested in state or probabilities could just access them from `lstm.state` or `lstm.probabilities?` That's perhaps even easier and less options (though maybe an anti-pattern).

Reply:
That will make the user make two calls: one for `probabilities` and one for `generate`. Looking at other ml5 classes, we have opted for returning a single object with probabilities whenever possible (`ImageClassification` and `Word2Vec` for instance)